src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -292,9 +293,14 @@ public void runDevAppServer1Command(List<String> jvmArgs, List<String> args)
 
     logCommand(command);
 
-    Map<String, String> environment = Maps.newHashMap();
-    environment.put("JAVA_HOME", javaHomePath.toAbsolutePath().toString());
-    processRunner.setEnvironment(environment);
+    Map<String, String> devServerEnvironment = Maps.newHashMap(environment);
+    if (!devServerEnvironment.isEmpty()) {
+      logger.info("Setting appengine-web.xml configured environment variables: "
```

Comment:
no, you're right. I was planning on improving this in the other PR (that allows client supplied env vars) since we'll have to differentiate between them. But yea, I think maybe moving them to the other class is the right approach.